src/PipelinesPage.tsx

```diff
@@ -54,21 +54,28 @@ const PipelineCard = ({
   pipeline,
   onInstantiate,
   showLink,
-}: PipelineCardProps): React.ReactElement => (
-  <Card className="mb-4 h-100">
-    <Card.Header as="h5">
-      <Link to={showLink}>{pipeline.name}</Link>
-    </Card.Header>
-    <Card.Body className="d-flex flex-column">
-      <Card.Text>{pipeline.description}</Card.Text>
-      <div className="mt-auto d-flex flex-column flex-md-row">
-        <Button variant="primary" onClick={onInstantiate}>
-          Instantiate
-        </Button>
-      </div>
-    </Card.Body>
-  </Card>
-);
+}: PipelineCardProps): React.ReactElement => {
+  const astarte = useAstarte();
+  return (
+    <Card className="mb-4 h-100">
+      <Card.Header as="h5">
+        <Link to={showLink}>{pipeline.name}</Link>
```

Comment:
We should render the name as a label instead of a Link when the permission is missing:

```tsx
astarte.token?.can('flow', 'GET', `/pipelines/${pipeline.name}`)
```